src/Features/Core/Portable/Navigation/INavigableDocumentLocation.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Navigation
+{
+    internal interface INavigableLocation
+    {
+        /// <summary>
+        /// Navigates to a location opening or presenting it in a UI if necessary.  This work must happen quickly. Any
+        /// expensive async work must be done by whatever component creates this value. This method is async only to
+        /// allow final clients to call this from a non-UI thread while allowing the navigation to jump to the UI
+        /// thread.
+        /// </summary>
+        Task<bool> NavigateToAsync(CancellationToken cancelletionToken);
```

Comment:
```suggestion
        Task<bool> NavigateToAsync(CancellationToken cancellationToken);
```

Reply:
wat.  how did that happen.

Reply:
this si actually fixed in the next PR :)